packages/@sanity/cli/src/commands/functions/testFunctionsCommand.ts

```diff
@@ -1,76 +1,96 @@
 import {type CliCommandDefinition} from '../../types'
 
 const helpText = `
+Arguments
+  <name> The name of the Sanity Function
+
 Options
   --data <data> Data to send to the function
   --file <file> Read data from file and send to the function
-  --name <name> The name of your Sanity Function
   --timeout <timeout> Execution timeout value in seconds
+  --api <version> Sanity API Version to use
+  --dataset <dataset> The Sanity dataset to use
+  --project-id <id> Sanity Project ID to use
+
 
 Examples
   # Test function passing event data on command line
-  sanity functions test --name echo --data '{ "id": 1 }'
+  sanity functions test echo --data '{ "id": 1 }'
 
   # Test function passing event data via a file
-  sanity functions test -name echo --file 'payload.json'
+  sanity functions test echo --file 'payload.json'
 
   # Test function passing event data on command line and cap execution time to 60 seconds
-  sanity functions test -name echo --data '{ "id": 1 }' --timeout 60
+  sanity functions test echo --data '{ "id": 1 }' --timeout 60
 `
 
 const defaultFlags = {
-  data: undefined,
-  file: undefined,
-  name: '',
-  timeout: 5, // seconds
+  'data': undefined,
+  'file': undefined,
+  'timeout': 5, // seconds
+  'api': undefined,
```

Comment:
Also, something we should change in runtime CLI. If `process.env.SANITY_INTERNAL_ENV` is set to `staging` the API value should be `api.sanity.work` otherwise the default is `api.sanity.io` but we should still let folks override this if they provide a flag.

Reply:
This is a version like 2025-01-01 not a domain I don't think...

Reply:
I don't think it's the domain. Not vital for this release either way

Reply:
oh gawd, so sorry.